src/google/protobuf/compiler/java/java_helpers.cc

```diff
@@ -76,6 +76,16 @@ const char* kForbiddenWordList[] = {
   "class",
 };
 
+const std::set<std::string> kReservedNames = {
```

Comment:
for consistency, can you use string instead of std::string here and below.

Reply:
Sure, I also changed the `std::set` to `std::unordered_set` since you were already using it.